<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Karz-ı Hasen</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/karz-i_hasen.png</normaloff>../images/karz-i_hasen.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Sisteme Giriş</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="input_tc">
     <property name="inputMask">
      <string>99999999999</string>
     </property>
     <property name="placeholderText">
      <string>TC. Kimlik No.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="input_pass">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Şifre</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_giris">
     <property name="text">
      <string>Giriş</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
